backend/scripts/inject_course_build_sqs.py

```diff
@@ -0,0 +1,31 @@
+import json
+import argparse
+from rope.api import settings
+import boto3
+
+
+def inject_course_build(sqs_client, sqs_queue_name, course_build_id):
+    queue_url_data = sqs_client.get_queue_url(QueueName=sqs_queue_name)
+    queue_url = queue_url_data["QueueUrl"]
+    message_body = {
+        "course_build_id": course_build_id
+    }
```

Comment:
Let's re-use the helper function [here](https://github.com/openstax/rope/blob/ba1ee9561aaf7746e4409a351f1482abd2156e69/backend/api/utils.py#L37) for this.

Reply:
Updated code to use helper function.